charts/seed-bootstrap/charts/vpa/charts/runtime/templates/vpa-webhook-config.yaml

```diff
@@ -0,0 +1,51 @@
+{{- if .Values.admissionController.enabled }}
+{{- if not .Values.admissionController.registerWebhook }}
+---
+apiVersion: admissionregistration.k8s.io/v1
+kind: MutatingWebhookConfiguration
+metadata:
+  name: vpa-webhook-config
+webhooks:
+- admissionReviewVersions:
+  - v1beta1
+  clientConfig:
+    caBundle: {{ .Values.admissionController.caCert }}
+{{- if .Values.admissionController.registerByURL }}
+    url: https://vpa-webhook.{{.Release.Namespace}}:{{.Values.admissionController.servicePort}} # the port is only respected if register-by-url is true, that's why it's in this if-block
```

Comment:
I guess `{{ .Release.Namespace }}` is wrong here if this webhook config shall be deployed into the shoot